client/client.go

```diff
@@ -1,172 +0,0 @@
-package client
-
-import (
-	"fmt"
-	"io/ioutil"
-	"bytes"
-	"net/http"
-	"encoding/json"
-	"database/sql"
-	
-	_ "github.com/go-sql-driver/mysql"
-)
-
-type User struct {
-	UserID *int
-	Email  []string
-}
-
-func Check(userID int, emails []string) error {
```

Comment:
Where is this function being called? Also should it return a list of users that are in the bf?